CHANGELOG.md

```diff
@@ -148,6 +148,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* (baseapp) [#15303](https://github.com/cosmos/cosmos-sdk/pull/15303) Remove previous block data from context before PrepareProposal + add ChainID to baseapp in order to pass it in the context.
```

Comment:
Given that this code is currently unreleased, I don't think a changelog is necessary.